Makefile

```diff
@@ -39,29 +42,43 @@ generate_phantomjs:
 	cd ./nodePhantomjs && ./generate.sh $(VERSION)
 
 base: generate_base
-	cd ./base && docker build -t $(NAME)/base:$(VERSION) .
+	cd ./base && docker build $(BUILD_ARGS) -t $(NAME)/base:$(VERSION) .
```

Comment:
I don't see the `${BIULD_ARGS}` being used anywhere in a Dockerfile. Is this a forward looking change?